recipes/mysterymaster/run_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mysterymaster -h >/dev/null
```

Comment:
Please move those two lines in the 'test' section in meta.yaml and use the same improvement suggestion as for FindBarcodes